src/mode.js

```diff
@@ -59,15 +59,12 @@ function getMode_() {
       // occur during local dev.
       !!document.querySelector('script[src*="/dist/"],script[src*="/base/"]');
 
-  const overrideDevelopment = parseQueryString(location.hash)['development'];
-  const development = overrideDevelopment != undefined
-      ? overrideDevelopment == '1'
-      : !!document.querySelector('script[development]');
+  const developmentQueryString = parseQueryString(location.hash)['development'];
 
   return {
     localDev: isLocalDev,
     // Triggers validation
-    development: development,
+    development: developmentQueryString == 1,
```

Comment:
If all the same, I'd rather have a direct comparison, e..g `developmentQueryString == '1'`. In this case, the type will be coerced, but I simply do not see a reason to rely on coercion here  since the variable is initialized as string just two lines above.